cql3/statements/role-management-statements.cc

```diff
@@ -59,6 +59,7 @@
 #include "gms/feature_service.hh"
 #include "transport/messages/result_message.hh"
 #include "unimplemented.hh"
+#include "concrete_types.hh"
```

Comment:
(commenting on 391d1f2b210d5b157d63fee951d1d67fc068cb98 changelog)

I think we do need to update running sessions. Perhaps it can be in a follow-up, but we should follow-up quickly and make this easy to use instead of requiring a rolling restart.

Reply:
(and probably iterating over all connections is fine)